test/e2e/install_test.go

```diff
@@ -54,7 +54,8 @@ func (s *installTestSuite) TestInstallMinorVersionPin() {
 
 	s.assertPinnedInstallScript("7.42.0")
 
-	s.addExtraIntegration()
+	// Disabled : integrations expired since First May 2024 for 7.42.0
+	//s.addExtraIntegration()
```

Comment:
I think it's fine if we forget about checking this specific functionality since I don't think it's related to the system under test.

```suggestion
```